internal/audit/kafka/conf.go

```diff
@@ -23,10 +23,29 @@ const (
 	defaultMaxBufferedRecords = 250
 )
 
+type Authentication struct {
+	TLS *TLS `yaml:"tls"`
+}
+
+type TLS struct {
+	// CAPath is the path to the CA certificate.
+	CAPath string `yaml:"caPath" conf:"required,example=/path/to/ca.crt"`
+	// CertPath is the path to the client certificate.
+	CertPath string `yaml:"certPath" conf:"required,example=/path/to/tls.cert"`
+	// KeyPath is the path to the client key.
+	KeyPath string `yaml:"keyPath" conf:"required,example=/path/to/tls.key"`
```

Comment:
The cert and key paths are no longer required, so we can remove that from the struct tags. Note that here and for reload interval we need a leading comma (i.e. `,example=...`) to avoid that `<DEFAULT_VALUE_NOT_SET>` in the conf docs.